recipes/js2py/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "Js2Py" %}
+{% set version = "0.67" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 168952e3827198f68eacbf84a7b23c80a55794415291f821ec6c96f3fd9c3253
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pyjsparser >=2.5.1
+    - python
+    - six >=1.10
+    - tzlocal >=1.2
```

Comment:
Are all of these really build-time requirements?

Reply:
No they aren't. I just checked by running conda-build locally, and it only required python and pip. Cleaned this up.